oauth/tool/src/java/org/sakaiproject/oauth/servlet/AuthorisationServlet.java

```diff
@@ -171,6 +173,7 @@ private void sendToAuthorisePage(HttpServletRequest request, HttpServletResponse
 
             request.getRequestDispatcher(authorisePath).forward(request, response);
         } catch (OAuthException e) {
+            log.warn("Problem with oauth request_token: {}", e.getMessage());
```

Comment:
using e.toString() vs e.getMessage() will print the class plus and the message
```suggestion
            log.warn("Problem with oauth request_token: {}", e.toString());
```